be/src/exec/vectorized/schema_scan_node.cpp

```diff
@@ -14,11 +16,14 @@ namespace starrocks::vectorized {
 
 SchemaScanNode::SchemaScanNode(ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs)
         : ScanNode(pool, tnode, descs),
+          _tnode(tnode),
           _is_init(false),
           _table_name(tnode.schema_scan_node.table_name),
           _tuple_id(tnode.schema_scan_node.tuple_id),
           _dest_tuple_desc(nullptr),
-          _schema_scanner(nullptr) {}
+          _schema_scanner(nullptr) {
+    _name = "schema_scan";
+}
```

Comment:
_schema_scanner(nullptr),
_name(schema_scan) {
}

Reply:
> _schema_scanner(nullptr), _name(schema_scan) { }

_name is a member of the parent class ScanNode. It seems that we can't put it in the initialization list.